dist/LICENSE

```diff
@@ -174,6 +174,20 @@
       of your accepting any such warranty or additional liability.
 
 
+========================================================================
```

Comment:
@kezhenxu94 . I failed to include the license dep checking into CI due to the order of groups cannot stay the same. Skywalking-eye changed the content on rerunning `dep resolve`. Any thoughts?

Reply:
I also want to mention this issue since in the last PR #129 there's huge amount of changes due to the unstable order.

Reply:
We should not rerun `make license-dep` except for releasing for now.

Reply:
Should be fixed in https://github.com/apache/skywalking-eyes/pull/115